Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/BasicActionBuilder.java

```diff
@@ -36,6 +38,7 @@
 
     private Runnable runnable;
     private AzureActionEnum action;
+    private String doingName;
```

Comment:
Rename this parameter? like `actionName` ?

Reply:
It is not an Action Name, it is a action DOING name such as starting/stopping/etc.

Reply:
We may sync later for this parameter